webapp/templates/team/menu.html.twig

```diff
@@ -53,8 +53,15 @@
                 </li>
             {% endif %}
         </ul>
-
-        <div class="ms-auto me-3">
+        
+        <div class="ml-auto me-3">
+            <a class="nav-link d-none" id="notify_enable">
```

Comment:
We should also need the `notify_disable` link here right?

Reply:
I was thinking of during the contest there'll be no disadvantage for a team to enable notifications, so no such button implemented. I'll add it if you think its necessary.

Reply:
I would prefer teams to be able to enable/disable it. And if we show the enable button like here, we should also show the disable button.